### PR TITLE
Removed SegueClass, presentation and dismissal logic reworked

### PR DESCRIPTION
**Pull request by EgorSobko**
Presentation and dismissal logic reworked.

### CODE REVIEW

File: MenuExample/Controller/ViewController.swift [hunk 1]
```diff
@@ -24,6 +28,14 @@ class ViewController: UIViewController {
                 super.prepareForSegue(segue, sender: sender)
         }
     }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        menuAnimator = MenuTransitionAnimator(shouldPassEvents: false, tappedOutsideHandler: {
```
Comment (maximletushov) [inline MenuExample/Controller/ViewController.swift]: @EgorSobko does it make sense to rename 'shouldPassEvents' to 'shouldPassEventsOutsideMenu' ?

Comment (EgorSobko) [inline MenuExample/Controller/ViewController.swift]: Yeah, it would be more clear about meaning of this flag

**Comment by maximletushov** [comment]
@EgorSobko thank you. Could you plz add updates to README in this PR? Thanks.

**Comment by maximletushov** [comment]
@EgorSobko thanks :+1:

**Pull request merged**